auth-server/src/test/java/com/makeurpicks/service/PlayerServiceTest.java

```diff
@@ -1,28 +1,37 @@
 package com.makeurpicks.service;
 
-import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.runners.MockitoJUnitRunner;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.SpringApplicationConfiguration;
-import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.boot.test.autoconfigure.orm.jpa.AutoConfigureTestDatabase;
 import org.springframework.security.crypto.password.PasswordEncoder;
-import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 
-import com.makeurpicks.AuthServerApplication;
-import com.makeurpicks.domain.Player;
-import com.makeurpicks.domain.PlayerBuilder;
+import com.makeurpicks.dao.ClientDetailsDAO;
+import com.makeurpicks.domain.OAuthClientDetails;
 
 //@RunWith(SpringJUnit4ClassRunner.class)
 //@SpringApplicationConfiguration(classes = AuthServerApplication.class)
+@RunWith(MockitoJUnitRunner.class)
+//@DataJpaTest
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
 public class PlayerServiceTest {
```

Comment:
Hi Pawal, 
I did not work on Player service as it was not required for my feature myp47 and myp48. That is the reason I did not write test case for them. I can write if needed and If a new JIRA task is present.

Thanks
Hari